stores/moderationSubmissionsStore.ts

```diff
@@ -101,6 +101,21 @@ export const useModerationSubmissionsStore = defineStore(
             }
         }
 
+        async function approveSubmission() {
+            if (!selectedSubmissionId.value) {
```

Comment:
Unblocking: I think we should probably log an error if someone tries to approve a submission without a valid ID.

Reply:
I'll add that as well as toasts.

I can do that in this PR @theyokohamalife or a future one.

I wanted to add the toast notifications to yours as well.

Maybe after this one I can have one for these that adds the error messaging?

🤔

Reply:
@theyokohamalife Once this is merged I will make a ticket and add all the toast errors that show when incorrect actions are taken or there is failure.

Reply:
At the very least we can `console.error` here with the ID.